pkg/cmd/cli/cmd/create/route.go

```diff
@@ -409,3 +430,10 @@ func resolveRouteName(args []string) (string, error) {
 	}
 	return "", nil
 }
+
+func resolveWildcardPolicy(wildcardpolicy string) (string, error) {
+	if len(wildcardpolicy) > 0 && (wildcardpolicy != "Subdomain" && wildcardpolicy != "None") {
+		return "", fmt.Errorf("only \"Subdomain\" or \"None\" are supported for wildcardpolicy")
+	}
+	return wildcardpolicy, nil
+}
```

Comment:
Maybe this should use validation from the route api (we will need to `export` something out in the route validation for that though). This looks good for now but if you can please add a comment here that we should use the validation from the route.  Thx